src/Yaml/Form/Transformer/FormYamlRector.php

```diff
@@ -4,7 +4,7 @@
 
 use Rector\Core\Contract\Rector\RectorInterface;
 
-interface FormYamlTransformer extends RectorInterface
+interface FormYamlRector extends RectorInterface
```

Comment:
Naming best practise: anythign that inherits from `RectorInterface` should have `Rector` suffix.
It will the orientation in the code with united naming :+1:

Reply:
Agree. This was historically an in between step before we had the FileProcessors in core.